app/assets/javascripts/discourse/app/helpers/html-safe.js

```diff
@@ -1,5 +1,13 @@
 import { htmlSafe as emberHtmlSafe } from "@ember/template";
+import deprecated from "discourse/lib/deprecated";
 
 export default function htmlSafe(string) {
+  deprecated(
```

Comment:
This is going to fire for anyone still using `.hbs`, and they'll have no way to resolve it.

How do you think about keeping this as a 'linting-only deprecation', until we eventually deprecate hbs?